porch/hack/create-deployment-blueprint.sh

```diff
@@ -118,6 +125,16 @@ function customize-image {
     "newTag=${TAG}"
 }
 
+function customize-image-in-command {
+  local OLD="${1}"
+  local NEW="${2}"
+
+  kpt fn eval "${DESTINATION}" --image search-replace:v0.2.0 -- \
+	  "by-path=spec.template.spec.containers[*].command[*]" \
+	  "by-value-regex=(.*)${OLD}" \
```

Comment:
[nit, since we work with a fairly constrained inputs]
the image name contains regex symbols (i.e. dot: `.`) is there a way to escape the value so it is matched literally?

Reply:
It's possible. I will fix this in follow-up PR